### ISSUE TITLE
Update page number when scrolling on "show-all" mode

### ISSUE THREAD

**Issue by anatel** [VadimDez/ng2-pdf-viewer#223]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
```
In "show-all" mode I want to display which page is currently displayed after scrolling to it, meaning I would expect the pageChange event to be emitted when scrolling to a new page.

**Comment by rttmax** [comment]
+1

**Comment by tomixtomi12** [comment]
+1

**Comment by eatmyball** [comment]
Is this issue been fixed?

**Comment by TimMeissner** [comment]
+1

**Comment by lastSeries** [comment]
@VadimDez, is this issue been solved? I hope this issue solved as soon as possible, because my programme have this funciton

**Comment by WhatzzUp** [comment]
+1

**Comment by houssamhammoud** [comment]
@VadimDez  is this issue been solved? I hope this issue solved as soon as possible, because my programme need this function?

**Comment by adhityan** [comment]
Was this issue resolved? Don't see a page scroll into view event in the documentation.

**Comment by jiangyh1024** [comment]
@ColinT

**Comment by jiangyh1024** [comment]
may I ask what does 'PR' mean ?  personal repository ?

**Comment by TomLaggna** [comment]
@ColinT created a PR based on your changes that fixes the remaining problems. #445

**Comment by VadimDez** [comment]
Released in `6.0.0` https://github.com/VadimDez/ng2-pdf-viewer/blob/master/CHANGELOG.md#600
